joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstForExpressionsCreator.scala

```diff
@@ -318,8 +318,11 @@ trait AstForExpressionsCreator(implicit withSchemaValidation: ValidationMode) {
         x.arguments.map(astForMethodCallArgument) :+ typeRef
     }
 
-    val constructorCall    = callNode(node, code(node), callName, fullName, DispatchTypes.DYNAMIC_DISPATCH)
-    val constructorCallAst = callAst(constructorCall, argumentAsts, Option(tmpIdentifier))
+    val constructorCall =
+      callNode(node, Defines.Initialize, Defines.Initialize, Defines.Any, DispatchTypes.DYNAMIC_DISPATCH)
```

Comment:
I suppose this is just nit-picking, but I don't like that the code field is simply `initialize`. Why not `code(node)`?

Reply:
Related to the node I left about parser packing being a bit weird on parenthesis-less calls, it would set undesired code e.g. `A` or `a = A` instead of `A.new`. If preferred, I could construct a code block from the target and arguments.

Reply:
Actually nevermind, I've changed it to `code(node)` and things seem fine

Reply:
perfect :)